admin/src/components/Service.js

```diff
@@ -1,5 +1,12 @@
 import axios from "axios";
 
+const tokenConfig = {
+  headers: {
+    Authorization: `Bearer ${window.localStorage.getItem("token")}`,
```

Comment:
tokent nem a legmegfelelobb igy tarolni foleg ezzel a nevvel a localstorage-ban. mivel ehhez minden weblap hozza fog ferni es ez egy eleg altalanos nev. @6uliver javaslatara jobb lenne auth service-t hasznalni.

Reply:
Nem fog más weblap hozzáférni, csak az admin website, mert domain alapján szerparálja a storage-et a böngésző, ahogy a sütiket is, így a név is rendben van. Az auth service alatt én csak kódkiszervezést értettem, hogy ne legyen mindenhol elszórva ez a window.localStorage.getItem("token"), hanem legyen egy AuthService.getToken meg AuthService.isLoggedIn metódus, ami elrejti az implementációt.